client/js/jsonedit.js

```diff
@@ -830,6 +881,23 @@ function jeShowCommands() {
   if(jeSecondaryWidget)
     commandText += `\n\n${jeSecondaryWidget}\n`;
   $('#jeCommands').innerHTML = commandText;
+  var buttons = [... document.querySelectorAll('#jeCommands>button')];
+  if (buttons.length > 0) {
+    buttons.forEach((e) => {
+      if (e != undefined) document.getElementById(e.id).addEventListener('click', clickButton);
+    });
+  }
+}
+
+const clickButton = function (event) {
+  jeCommands.find(o => o.id == event.currentTarget.id).call();
+  if (jeContext != 'macro') {
+    jeGetContext();
+    if(jeWidget && !jeJSONerror)
+      jeApplyChanges();
+    if (jeContext[0] == '###SELECT ME###')
+      jeGetContext();
```

Comment:
This doesn't look like it belongs here but I'm pretty sure it's fixing a bug that I also noticed before so whatever. :D

If I didn't miss anything this is only called for clicks and not hotkeys, right? Feels like something that should be the same either way.

Reply:
yes, clicks only. that bug didn't occur using keys